README.md

```diff
@@ -23,12 +23,12 @@ Kindly consider leaving a :star: if you like the repository and our organisation
 
 * Run termiboard
 
-```
+```console
 go run *.go
```

Comment:
This should be changed to ```go run .``` instead as we have test files now

Reply:
Okay sure